ops/gulp/docs-icons.js

```diff
@@ -0,0 +1,9 @@
+const gulp = require('gulp');
+
+const task = (cb) => {
+    return gulp.src('./src/styles/icons/**.svg')
```

Comment:
Pull this from the `dist` directory instead of `src`. For the most part, the docs shouldn't need much from the `src` directory, the main exception being the `src/templates/` includes used for the component pages.

Just like the `icons-build` is the pre-build step before the package is built, you should assume that the package is built and all the files required for the website are available in `dist` (or `tmp`) before the site Is built. The order must be 1) build icons, 2) build package, 3) build site.

This ensures there is no disconnect between the `src` and the `dist` versions. Check out the `docs-css` task.